addons/l10n_be/i18n/nl.po

```diff
@@ -317,7 +301,7 @@ msgstr "V Aftrekbaar"
 #. module: l10n_be
 #: model:account.report,name:l10n_be.tax_report_vat
 msgid "VAT Return"
-msgstr "Btw-aangifte"
+msgstr "BTW aangifte"
```

Comment:
Correct spelling is "Btw-aangifte"

Reply:
These are the translations from the latest Transifex project. If it's not accurate, you may want to fix it on Transifex directly